tasks/patches/add-sbom-and-push.yaml

```diff
@@ -102,6 +102,9 @@
     name: inject-sbom-and-push
     resources: {}
     script: |
+      # Expose base image digests
+      buildah images --format '{{ .Name }}:{{ .Tag }}@{{ .Digest }}' > $(results.BASE_IMAGES_DIGESTS.path)
```

Comment:
we should probably ignore temporary image which we created -> `| grep -v $(params.IMAGE)`